testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java

```diff
@@ -304,7 +303,7 @@ public void testShowSchemasLikeWithEscape()
             Set<Object> allSchemas = computeActual("SHOW SCHEMAS").getOnlyColumnAsSet();
             assertEquals(allSchemas, computeActual("SHOW SCHEMAS LIKE '%_%'").getOnlyColumnAsSet());
             Set<Object> result = computeActual("SHOW SCHEMAS LIKE '%$_%' ESCAPE '$'").getOnlyColumnAsSet();
-            assertNotEquals(allSchemas, result);
+            assertThat(result).isSubsetOf(allSchemas);
```

Comment:
> testShowSchemasLikeWithEscape assumes that there is at-least one schema
> without an `_` in 

there should always be a schema with underscore (information_schema)
and some schema without underscore, like default/public/tpch or something

what am i missing?

Reply:
btw testing `assertThat(result).isSubsetOf(allSchemas)` may fail in case of concurrent test execution: `result` may contain a schema that did not exist when `allSchemas` was being computed

Reply:
It doesn't always have to be the case. e.g. Some catalogs have test data under schemas like test_schema and no public schema (or any other without underscore exists).

Note that entire assertion is wrapped within assertEventually (pre-existing with no good solution).

Reply:
I can fix the catalogs themselves but it's an unwritten rule.

I'm also fine with just adding a comment here about the assumptions. (Or adding a `verify` that verifies at-least one non-underscore schema exists and fails with instructions on what to do).

Reply:
> Or adding a verify that verifies at-least one non-underscore schema exists and fails with instructions on what to do).


works for me. 
in the worst case, a connector unable to create test schema without undescrore will... override this method. not a big deal.

Reply:
maybe combine the two:

```
assertThat(result)
  .isSubsetIOf(allSchemas)
  .isNotEqualTo(allSchemas);
``

Reply:
Nice.